Updated the flash function

Paul3816547290 opened the pull request:
Now the caller can control the color, number of flashes, and duration between flashes

Paul3816547290 committed:
Update element.rb

Paul3816547290 commented:
https://github.com/watir/watir#note-on-patchespull-requests
Suggests I should write tests - happy to do that but dont know where these are or how they are executed

p0deje commented:
Thank you for pull request!

> Suggests I should write tests - happy to do that but dont know where these are or how they are executed

I think this is fine assuming you've added new example in documentation which is automatically tested.

p0deje commented on lib/watir/elements/element.rb:
```diff
@@ -213,26 +213,28 @@ def drag_and_drop_by(right_by, down_by)
     #
 
-    def flash
+    def flash(color="yellow",flashes=5,delay=0.1)
```
Let's use keyword arguments so that you can pass just one of them (e.g. you only want to control number of flashes):

```ruby
def flash(color: 'yellow', flashes: 5, delay: 0.1)
  ...
end
```

p0deje replied:
Also, let's keep old defaults:

* flashes - 10
* delay - 0

p0deje commented on lib/watir/elements/element.rb:
```diff
@@ -213,26 +213,28 @@ def drag_and_drop_by(right_by, down_by)
+    # Flashes (change background color to a new color and back a few times) element.
     #
-    # @example
+    # @examples
```
Do not change it

p0deje commented on lib/watir/elements/element.rb:
```diff
@@ -213,26 +213,28 @@ def drag_and_drop_by(right_by, down_by)
-    # @example
+    # @examples
     #   browser.text_field(name: "new_user_first_name").flash
+    #   browser.text_field(name: "new_user_first_name").flash("green",3,0.05)
```
Please, keep the code style:

```ruby
browser.text_field(name: "new_user_first_name").flash("green", 3, 0.05)
```

p0deje reviewed (changes requested):
Overall it looks great, thank you for contribution.

However, let's fix few things before we merge this.

Paul3816547290 committed:
Updated flash function parameters and defaults

Flash function can be called with named parameters, and the defaults are the same as before the change, examples expanded

Paul3816547290 commented:
All amendments suggested completed and remerged

p0deje commented:
Merged in c30132eb46f570fb50d70c96de9e977ba9e4772c, thank you for contribution!
